habitat-lab/habitat/config/habitat/dataset/rearrangement/zxz_manipulation.yaml

```diff
@@ -6,5 +6,6 @@ defaults:
 
 type: RearrangeDataset-v0
 split: eval
-data_path: data/datasets/manipulation/manipulation_eval.json.gz
+#data_path: data/datasets/single_agent_eval.json.gz
+data_path: data/datasets/single_agent_success.json.gz
```

Comment:
Where are those files? I did not see them in google drive.

Reply:
These are for locally test, new yamls and datasets have been uploaded on drive.